### ISSUE TITLE
[Feature]Can i clone SymbolicRegression.jl manually while running pysr.install()?

### ISSUE THREAD

**Issue by WhiteGL** [MilesCranmer/PySR#293]
I always met difficulties about cloning git repos while running pysr.install(). But i can clone those repos manually. So can i put these repos like SymbolicRegression.jl somewhere so that i can skip the step of cloning while runing install()?

**Comment by MilesCranmer** [comment]
Yes, the `julia_project` parameter is for this purpose. Point it to the location of your local git repo.

https://astroautomata.com/PySR/api/#environment
